Bottom: d3df3fcac2232075b82412e3d0babf6b58e6fb39
Top:    aa3b998e29f9bb2386214ab67d03a73ec9bf92bd
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-24 22:44:07 +0000

Invalidate board contour when a via / pin is altered

The drill-holes of vias and pins can affect the board contour


---

diff --git a/src/polygon.c b/src/polygon.c
index c1f78c1..a9b0f29 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -1812,6 +1812,10 @@ ClearFromPolygon (DataType * Data, int type, void *ptr1, void *ptr2)
           strcmp (layer->Name, "route") == 0)
         Data->outline_valid = false;
     }
+  else if (type == PIN_TYPE || type == VIA_TYPE)
+    {
+        Data->outline_valid = false;
+    }
 
   if (type == POLYGON_TYPE)
     InitClip (PCB->Data, (LayerType *) ptr1, (PolygonType *) ptr2);
